mmdet/datasets/xml_style.py

```diff
@@ -25,9 +25,12 @@ class XMLDataset(BaseDetDataset):
     def __init__(self,
                  img_subdir: str = 'JPEGImages',
                  ann_subdir: str = 'Annotations',
+                 file_client_args: dict = dict(backend='disk'),
```

Comment:
Unnecessary code, file_client_args will be setting in `BaseDetDataset`, please have a check